apps/test/unit/craft/utilsTest.js

```diff
@@ -0,0 +1,57 @@
+// We have to include the locale files below as translations must be loaded in the global
+// scope for HeadlessChrome to run properly.
+import commonI18n from '@cdo/locale'; // eslint-disable-line no-unused-vars
+import craftI18n from '@cdo/apps/craft/locale'; // eslint-disable-line no-unused-vars
+
+import {expect} from '../../util/reconfiguredChai';
+import sinon from 'sinon';
+import * as utils from '@cdo/apps/craft/utils';
+import * as craftRedux from '@cdo/apps/craft/redux';
+
+describe('craft utils', () => {
+  describe('handlePlayerSelection', () => {
+    const defaultPlayer = 'Alex';
+
+    beforeEach(() => {
+      sinon.stub(craftRedux, 'closePlayerSelectionDialog');
+    });
+
+    afterEach(() => {
+      sinon.restore();
+    });
+
+    it('closes player selection dialog after opening', () => {
```

Comment:
nit: closes the dialog after selecting a player? this test name sort of makes it sound like it opens then closes immediately